src/osmf/lib/HDSPlayer.as

```diff
@@ -306,7 +308,11 @@ package
           dispatchEvent(new DynamicEvent(DynamicEvent.PAUSED,null));
           break;
         case MediaPlayerState.BUFFERING:
-          dispatchEvent(new DynamicEvent(DynamicEvent.BUFFERING,null));
+          {
```

Comment:
I think the default style of this file does not add these brackets for case statements.